coinbaseadvanced/client_websocket.py

```diff
@@ -39,7 +55,7 @@ def _create_message(self, message_type: str, product_ids: list, channel: str) ->
             "timestamp": int(time.time())
         }
 
-    def _handle_message(self, ws, message: str):
+    def _handle_message(self, ws: websocket.WebSocket, message: str) -> None:
```

Comment:
It seems this function is expected to return `None` no matter what. So, I slightly updated the code to reflect that.